onnxruntime/core/providers/webnn/builders/impl/squeeze_unsqueeze_op_builder.cc

```diff
@@ -80,46 +81,69 @@ Status UnsqueezeOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
     options.set("axes", emscripten::val::array(axes_data));
   }
 
-  emscripten::val output = model_builder.GetBuilder().call<emscripten::val>("unsqueeze", input, options);
+  emscripten::val output = emscripten::val::undefined();
+  if (op_type == "Squeeze") {
+    output = model_builder.GetBuilder().call<emscripten::val>("squeeze", input, options);
+  } else if (op_type == "Unsqueeze") {
+    output = model_builder.GetBuilder().call<emscripten::val>("unsqueeze", input, options);
+  } else {
+    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT,
+                           "SqueezeUnsqueezeOpBuilder::AddToModelBuilderImpl, unknown op: ", op_type);
+  }
+
   model_builder.AddOperand(node.OutputDefs()[0]->Name(), std::move(output));
   return Status::OK();
 }
 
 // Operator support related.
 
-bool UnsqueezeOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializers,
-                                           const Node& node,
-                                           const WebnnDeviceType /* device_type */,
-                                           const logging::Logger& logger) const {
+bool SqueezeUnsqueezeOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializers,
+                                                  const Node& node,
+                                                  const WebnnDeviceType /* device_type */,
+                                                  const logging::Logger& logger) const {
+  const auto& op_type(node.OpType());
   const auto& input_defs = node.InputDefs();
```

Comment:
```suggestion
  const auto& op_type = node.OpType();
  const auto& input_defs = node.InputDefs();
```

Just for consistency - it looks weird using two different initialization styles adjacent to each other (and just the slightest bit easier to read with fewer nested parens).